app/controllers/TransferCompleteController.scala

```diff
@@ -6,13 +6,29 @@ import configuration.KeycloakConfiguration
 import javax.inject.Inject
```

Comment:
Needs a line space between `package controllers` and `import java.util.UUID` statements at the beginning of the file